ajax.php

```diff
@@ -79,8 +113,10 @@
 			if ($oObj !== null)
 			{
 				$oDoc = $oObj->Get($aData['sBlobAttCode']);
-				$sFileName = $oDoc->GetFileName();
-				$sIcon = utils::GetAbsoluteUrlAppRoot().AttachmentPlugIn::GetFileIcon($sFileName);
+				// The unmodified filename is needed first to choose the icon right after, then we make right label
+				$sAttachmentLabel = $oDoc->GetFileName();
+				$sIcon = utils::GetAbsoluteUrlAppRoot().AttachmentPlugIn::GetFileIcon($sAttachmentLabel);
```

Comment:
I would revert this part as the `$sAttachmentLabel` gets overwritten anyway.
```suggestion
				$sFileName = $oDoc->GetFileName();
				$sIcon = utils::GetAbsoluteUrlAppRoot().AttachmentPlugIn::GetFileIcon($sFileName);
```